### ISSUE TITLE
wish: to get note text from stdin

### ISSUE THREAD

**Issue by johnd0e** [andyforceno/shuttle#1]
Hi
I want to send multiline notes.
I guess it would be easier if we can use stdin
Thanks!

**Comment by andyforceno** [comment]
Hi! Yes, this is something I have wanted, as well. I've been having some difficulty getting Pushbullet to accept multi-line text in general, most likely due to how it is handled in JSON. How Bash handles newlines may also play a role.

I like to have all the Linux devices on my network send smartctl test reports, among other things, all of which have newlines in them. Pushbullet always sends these truncated at the first newline, unless sent as a file, rather than a note. I would like to be able to do something like `cat text | shuttle -p -n device "title" "<stdin>"; instead. I'll start working on that again in the near future.

**Comment by fbartels** [comment]
Pushbullet-bash already allows to pass the message body over stdin. You can have a look at https://github.com/Red5d/pushbullet-bash/blob/master/pushbullet#L386-L391 for some inspiration. 

When reimplementing this I would not use <stdin> as a keyword to replace by the actual stdin, but would go a more unix'y way of passing "-" to indicate that this portion should be read from stdin instead.

**Comment by andyforceno** [comment]
@fbartels Thank you for the suggestion. I ended up using some of your code, namely `body=$(cat)`.  I had found some other suggestions on StackExchange, but yours seemed the most simple.

@johnd0e I'll make a commit tonight which adds the functionality you've requested. It will work like this: 

`command | shuttle -p -n device "This is the title"` with the output of the piped command being sent as the body. I only wish the Pushbullet browser extension provided more horizontal space, things like log files  or certain outputs (`df -ah`) are word-wrapped or difficult to parse. Otherwise, I'm pleased with the results.
